engine/runtime/src/test/java/org/enso/interpreter/test/NumbersTest.java

```diff
@@ -0,0 +1,94 @@
+package org.enso.interpreter.test;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import org.enso.polyglot.MethodNames;
+import org.graalvm.polyglot.Context;
+import org.junit.AfterClass;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
```

Comment:
@Akirathan, originally I was using `@Theories`, but after reading this [stackoverflow post](https://stackoverflow.com/questions/8779982/why-does-junit-run-test-cases-for-theory-only-until-the-first-failure):
> A Theory is wrong if a single test in it is wrong, according to the definition of a Theory. If your test cases don't follow this rule, it would be wrong to call them a "Theory".

and after realizing _theories_ are in `experimental` package, I [decided to switch to @Parametrized](https://github.com/enso-org/enso/pull/7796/commits/0ab53f9534ba1dea2485d74d2c904d56d6ed5845). The way to specify `parameters()` isn't as nice as in case of theories, but unlike the [academical theory activists](https://stackoverflow.com/a/13754931) I believe running all the test is what real developers want. I seem to like proper error reports more then purity.

Don't you want to migrate to `@Parametrized` too?